webpack/webpack.config.js

```diff
@@ -0,0 +1,92 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+module.exports = {
+  entry: './app/upload.js',
+  output: {
+    filename: 'upload.js',
+    path: '../src/collective/upload/browser/static',
```

Comment:
maybe the error was caused by the way they ran the command; you don't have to run that command at all, @idgserpro.